.github/workflows/ci_tests.yml

```diff
@@ -60,6 +54,25 @@ jobs:
       run: |
         echo "DOCKER_IMAGE_TAG=$(jq '.build.args.GEOS_TPL_TAG' -r .devcontainer/devcontainer.json)" >> "$GITHUB_OUTPUT"
 
+  get_current_labels:
```

Comment:
Can we look into manual triggers options instead of using labels? People usually add the labels and then leave them there forever. I think that, most of the times, one just wants to run the cuda builds once, and if they pass, probably does not need to rerun them any more until the very last second. So, I wonder if having an actual button to trigger  those jobs could be a good solution. I think it would save a lot of useless builds. Considering that we only have 2 runners I think it does matters.

Reply:
I believe things should now be separated as suggested. Note that now the job that checks for a specific label (ready to be merged or run integrated tests) now passes when that label is found. Thus it doesnt pick up when a label is removed, for example, but Im not sure if that happens often in the workflow.

Manual triggers are also an option - I have no idea how they work and how/if it would work with the all tests passed check (i.e. can you pass this if you only manually trigger one job, but all of the others passed previously)

Reply:
@CusiniM Do you know what GHA provides in terms of manual run?